### DISCUSSION TITLE
When Importing via RSS it would be ideal if posts would import into the user's editor of choice

### DISCUSSION THREAD

**Discussion by michael-tharrington** [forem/forem#16623] [Feature Requests]
Currently, if a user imports an article via RSS from `/settings/extensions` > **Publishing to ______ from RSS**, then the article always imports into the **Basic markdown** editor, even if the user has set their editor preference as **Rich + text markdown**. This can be a confusing experience for folks that aren't as familiar with the basic markdown editor. We've heard from some contributors who didn't realize how to post their articles after importing them this way.

Users can select their editor of choice from `/settings/customization` > **Writing**.

Semi-related: there's also a closed, but [highly-demanded request for organizations to be able to own their RSS Feed Import](https://github.com/forem/forem/issues/3363).

**Comment by Emery7164** [comment]
What are the alternative way of it. As we are also using the RSS method for [Instagram Pro APK](https://apkfilla.com/instagram-pro-apk/) importing. As someone told me that there is bundles of tools which you could but I am not finding those tools. Have you ever experience such tools. Let me know so I could try them.
